frontend/src/metabase/css/core/text.css

```diff
@@ -61,7 +61,7 @@
     .xl-text-right { text-align: right; }
 }
 
-.text-uppercase, :local(.text-uppercase) { text-transform: uppercase; }
+.text-uppercase, :local(.text-uppercase) { text-transform: uppercase; letter-spacing: 0.8px;}
```

Comment:
Only thing I'm thinking through as I look at this is we can theoretically apply the `.text-uppercase` class to any element with any size so there may be cases where this letter spacing value becomes somewhat unexpected.